opsfiles/enable-syslog.yml

```diff
@@ -0,0 +1,152 @@
+
+# variables
+- type: replace
+  path: /variables/name=ingestor_syslog_client_tls?
+  value:
+    name: ingestor_syslog_client_tls
+    options:
+      alternative_names:
+      - logs-opensearch.ingestor.service.cf.internal
+      ca: opensearch_ca
+      common_name: logs-opensearch.ingestor.service.cf.internal
+      extended_key_usage:
+      - client_auth
+    type: certificate
+    update_mode: converge
+
+- type: replace
+  path: /variables/name=ingestor_syslog_server_tls?
+  value:
+    name: ingestor_syslog_server_tls
+    options:
+      alternative_names:
+      - logs-opensearch.ingestor.service.cf.internal
+      ca: opensearch_ca
+      common_name: logs-opensearch.ingestor.service.cf.internal
+      extended_key_usage:
+      - server_auth
+    type: certificate
+    update_mode: converge
+
+- type: replace
+  path: /variables/name=archiver_syslog_client_tls?
+  value:
+    name: archiver_syslog_client_tls
+    options:
+      alternative_names:
+      - logs-opensearch.archiver.service.cf.internal
+      ca: opensearch_ca
+      common_name: logs-opensearch.archiver.service.cf.internal
+      extended_key_usage:
+      - client_auth
+    type: certificate
+    update_mode: converge
+
+- type: replace
+  path: /variables/name=archiver_syslog_server_tls?
+  value:
+    name: archiver_syslog_server_tls
+    options:
+      alternative_names:
+      - logs-opensearch.archiver.service.cf.internal
+      ca: opensearch_ca
+      common_name: logs-opensearch.archiver.service.cf.internal
+      extended_key_usage:
+      - server_auth
+    type: certificate
+    update_mode: converge
+
+- type: replace
+  path: /variables/name=archiver_syslog_client_tls?
+  value:
+    name: archiver_syslog_client_tls
+    options:
+      alternative_names:
+      - logs-opensearch.archiver.service.cf.internal
+      ca: opensearch_ca
+      common_name: logs-opensearch.archiver.service.cf.internal
+      extended_key_usage:
+      - client_auth
+    type: certificate
+    update_mode: converge
+
+- type: replace
+  path: /variables/name=archiver_syslog_server_tls?
+  value:
+    name: archiver_syslog_server_tls
+    options:
+      alternative_names:
+      - logs-opensearch.archiver.service.cf.internal
+      ca: opensearch_ca
+      common_name: logs-opensearch.archiver.service.cf.internal
+      extended_key_usage:
+      - server_auth
+    type: certificate
+    update_mode: converge
+
+
+# add in routes
```

Comment:
Are we sure we need the registered routes? Isn't using BOSH DNS aliases to point at the instances enough?

Reply:
In fact, I'm pretty sure we don't need or want these. Because having a registered route exposes these URLs to the public internet, while we only need these URLs for internal traffic and don't want them accessible from the public internet